rest_framework/fields.py

```diff
@@ -1227,6 +1227,8 @@ def to_internal_value(self, value):
         self.fail('invalid', format='[DD] [HH:[MM:]]ss[.uuuuuu]')
 
     def to_representation(self, value):
+        if isinstance(value, six.string_types):
+            return value
```

Comment:
This code just following same logic for TimeField:
https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/fields.py#L1199